documentation/book/con-what-the-cluster-operator-does.adoc

```diff
@@ -15,9 +15,15 @@ On {OpenShiftName} such a cluster can be deployed using the Source2Image feature
 image::cluster_operator.png[Cluster Operator]
 
 When the Cluster Operator is up, it starts to _watch_ for certain {ProductPlatformName} resources containing the desired Kafka or Kafka Connect cluster configuration.
-A `Kafka` resource is used for Kafka cluster configuration, and a `KafkaConnect` resource is used for Kafka Connect cluster configuration.
+By default it watches only in the same namespace or project where it is installed.
+The Cluster Operator can be configured to watch for more {Namespaces}.
+The resources Cluster Operator is watching for are:
 
-When a new desired resource (that is, a `Kafka` or `KafkaConnect` resource) is created in the {ProductPlatformName} cluster, the operator gets the cluster configuration from the desired resource and starts creating a new Kafka or Kafka Connect cluster by creating the necessary other {ProductPlatformName} resources, such as StatefulSets, Services, ConfigMaps, and so on.
+* A `Kafka` resource for Kafka cluster.
+* A `KafkaConnect` resource for Kafka Connect cluster.
+* A `KafkaConnectS2I` resource for Kafka Connect cluster with Source2Image support.
+
+When a new desired resource (that is, a `Kafka`, `KafkaConnect`, or `KafkaConnectS2I` resource) is created in the {ProductPlatformName} cluster, the operator gets the cluster description from the desired resource and starts creating a new Kafka or Kafka Connect cluster by creating the necessary other {ProductPlatformName} resources, such as StatefulSets, Services, ConfigMaps, and so on.
```

Comment:
When a new desired resource (that is, a `Kafka`, `KafkaConnect`, or `KafkaConnectS2I` resource) is  > When a new `Kafka`, `KafkaConnect`, or `KafkaConnectS2I` resource is created